info.yml

```diff
@@ -74,6 +74,15 @@ configuration:
           - [task, is_not, null]
           - [entity, is_not, null]
 
+    history_published_file_filters:
```

Comment:
Does this new setting need documentation?

Reply:
To me, it is clear and pretty straight forward what it does from the config setting description.. but that is just my opinion